backend/pick-git/src/main/java/com/woowacourse/pickgit/config/auth_interceptor_regester/scanner/package_scanner/SourceVisitor.java

```diff
@@ -0,0 +1,75 @@
+package com.woowacourse.pickgit.config.auth_interceptor_regester.scanner.package_scanner;
+
+import java.io.IOException;
+import java.nio.file.FileVisitResult;
+import java.nio.file.FileVisitor;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class SourceVisitor implements FileVisitor<Path> {
+
+    private final List<String> classPaths;
+
+    public SourceVisitor() {
+        this(new ArrayList<>());
+    }
+
+    public SourceVisitor(List<String> classPaths) {
```

Comment:
```suggestion
    private SourceVisitor(List<String> classPaths) {
```

부생성자가 public으로 열려 있으니까 주생성자는 private으로 닫는 게 어떨까요?

Reply:
주 생성자 또한 생성자 초기화를 위해 열어두는게 맞다고 생각됩니다.
해당 생성자는 테스트에서 사용되고 있습니다.